kernel/aster-nix/src/device/tdxguest/mod.rs

```diff
@@ -57,30 +56,24 @@ impl From<TdCallError> for Error {
 }
 
 impl FileIo for TdxGuest {
-    fn read(&self, buf: &mut [u8]) -> Result<usize> {
+    fn read(&self, _buf: &mut [u8]) -> Result<usize> {
         return_errno_with_message!(Errno::EPERM, "Read operation not supported")
     }
 
-    fn write(&self, buf: &[u8]) -> Result<usize> {
+    fn write(&self, _buf: &[u8]) -> Result<usize> {
         return_errno_with_message!(Errno::EPERM, "Write operation not supported")
     }
 
-    fn ioctl(&self, cmd: IoctlCmd, arg: usize) -> Result<i32> {
+    fn ioctl(&self, cmd: IoctlCmd, _arg: usize) -> Result<i32> {
         match cmd {
             IoctlCmd::TDXGETREPORT => {
-                let tdx_report: TdxReportRequest = read_val_from_user(arg)?;
-                match get_report(&tdx_report.tdreport, &tdx_report.reportdata) {
-                    Ok(_) => {}
-                    Err(err) => return Err(err.into()),
-                };
-                write_val_to_user(arg, &tdx_report)?;
-                Ok(0)
+                todo!()
```

Comment:
Why is `get_report` disabled?

Reply:
A DMA-based refactoring will be submitted soon.